spec/views/gsis/show.html.slim_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+require 'helpers/json_format_helper'
+
+describe 'gsis/show' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.